src/containers/Stewardship/StewardshipContent.tsx

```diff
@@ -1,59 +1,59 @@
-/* eslint-disable max-len */
-import React from 'react';
-import ELCALogo from '../../components/elcaLogo';
+// /* eslint-disable max-len */
+// import React from 'react';
+// import ELCALogo from '../../components/elcaLogo';
```

Comment:
@kathleenahall this code here can remain uncomented out please, it will never be visible